### ISSUE TITLE
Multiple input calendars issue

### ISSUE THREAD

**Issue by adrianwoj** [uvarov-frontend/vanilla-calendar#87]
Hi. 
Some context first - for my current project I need a simple calendar with option to set the locale based on the language of the application. After some initial tests it seems like vanilla-calendar is a good solution.

However - in version 2.6.0 (i checked a few earlier ones, and the issue was also there) - it seems like, when I try to use more than one calendar with two input fields, the closing of the calendar (when you click-away) only works with the last initialized one. Simple reproduction can be done with the following code:

```html
<input id="calendar-input1" type="text" placeholder="Choose date" readonly />
<input id="calendar-input2" type="text" placeholder="Choose date" readonly />
```

```js
const calendar = new VanillaCalendar('#calendar-input1', {input:true});
const calendar2 = new VanillaCalendar('#calendar-input2', {input:true});

calendar.init();
calendar2.init();
```

The current behaviour produces the following result:
![image](https://github.com/uvarov-frontend/vanilla-calendar/assets/11557531/3ab5b354-6d43-422a-9175-34a64ca8c34d)
^ 1 - click on the input field shows the calendar, 2 - click away **doesn't hide** the calendar
![image](https://github.com/uvarov-frontend/vanilla-calendar/assets/11557531/81514e63-a93d-4983-a3c7-79cf6939ec3e)
^ 3 - click on the second input shows the second calendar, 4 - click away hides only the second calendar.

I tried to create a CodeSandbox, hopefully it will work - https://codesandbox.io/s/vigilant-kate-fh4vdr?file=/calendar.js

Let me know if there is any solution for this issue :) Thanks!

**Comment by uvarov-frontend** [comment]
Hi, thanks for the problem report. The fix will be released in v2.6.1 in the coming days.
You need to either wait or write your own function to display a static calendar without the `{input:true}` parameter.

**Comment by adrianwoj** [comment]
Awesome, thanks!

**Comment by uvarov-frontend** [comment]
Issue fixed in v2.6.1
